addons/project/static/tests/tours/project_task_history.js

```diff
@@ -115,6 +115,35 @@ registry.category("web_tour.tours").add("project_task_history_tour", {
     }, {
         content: "Go back to projects view. this step is added because Tour can't be finished with an open form view in edition mode.",
         trigger: 'a[data-menu-xmlid="project.menu_projects"]',
+    }, {
+        content: "Open tab Tasks.",
+        trigger: 'button[data-menu-xmlid="project.menu_project_management"]',
+    }, {
+        content: "Open tab All Tasks.",
+        trigger: 'a[data-menu-xmlid="project.menu_project_management_all_tasks"]',
+    }, {
+        content: "Create a new task.",
+        trigger: 'button.o_list_button_add',
+    }, {
```

Comment:
Don't understand why it says this step fail, i add `watch=True` and see that the new task has been created successfully

Reply:
when i see the screenshot in the runbot (screenshot for the failing step) 
![image](https://github.com/odoo/odoo/assets/64937092/40036ecf-f9e3-4d66-b2c8-a44276a8f756)

what about adding the data in the py test manually in the runbot and start the tour manually, if everything is going well,

maybe saving the form and moving to step content: "Go back to projects view. this step is added because Tour can't be finished with an open form view in edition mode.", is take some time, so the step to navigate to all tasks doesn't find it, so try:
- remove the projects step return and move directly from the form to all tasks
or
- add some timeout to be sure the save was done and you're sure the view loaded, 

what do you think ?

Reply:
@RaoufGhrissi Adding data in py doesn't reflect what i 've fixed sir, anyway it still failed, what can go wrong ? :((

Reply:
hello @duong77476-viindoo  sorry, i was off for two weeks, i will take a look